_beats/reviewdog.yml

```diff
@@ -1,3 +0,0 @@
-runner:
```

Comment:
this doesn't seem to break our golint target like I expect it should.

Reply:
The reason why it didn't break is because the actual reviewdog cmd is run with the `-f` param: 

_beats/libbeat/scripts/Makefile:
```
REVIEWDOG?=reviewdog -conf ${ES_BEATS}/reviewdog.yml
```

Makefile:
```
GOLINT_COMMAND=$(shell $(GOLINT) ${GOLINT_TARGETS} | $(REVIEWDOG) -f=golint -diff="git diff 
```

However, I assume we want to use the config file, so I'll add it again. This file is still present in beats. It is not present in APM Server master. This is because it is missing in the `update_beats.sh` script. It looks like it had been added once and with the next beats update it was automatically removed again.

Reply:
I pushed an update that brings back this file.